arm-sql/2015-05-01/swagger/sql.core.json

```diff
@@ -0,0 +1,184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/move": {
```

Comment:
Remove it :), making it active so that you can look at this again, looks like you missed it

---
In reply to: [83115825](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r83115825) [](ancestors = 83115825,83113093,82919111,82916426)